tests/Stampie/Tests/MailerTest.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Stampie\Tests;
 
+
+use GuzzleHttp\Psr7\Response;
```

Comment:
Is it better to use Guzzle instead of https://github.com/php-http/mock-client in tests?